### ISSUE TITLE
[Feature request] Organise folders in collection by drag and drop

### ISSUE THREAD

**Issue by jaktestowac** [usebruno/bruno#1568]
Currently, folders within a Collection are sorted alphabetically. Users should have the ability to reorder the folders by dragging and dropping them into their preferred arrangement.

For example, similar to Postman, I want to move the `Operational` folder so it comes after the `Users` folder.
![obraz](https://github.com/usebruno/bruno/assets/72373858/9bc056cb-cd42-4203-bcb5-e120fa604d89)

To clarify how nesting folders appears:
![obraz](https://github.com/usebruno/bruno/assets/72373858/d095ea9b-b1ee-44c6-937a-83e25e2a1c0d)

Additionally, when importing a Collection from Postman, it should be organized in Bruno to maintain the same order as in Postman.
Mentioned here: https://github.com/usebruno/bruno/issues/1456

**Comment by DanaEpp** [comment]
I agree with this. I actually don't see a way to reorder this, even through manually editing config file.

I'd recommend you add a section in `bruno.json` that allows you to set the custom order of the folders within a collection, and then honor that in the UI. This way, when the runner is executed, it will follow that order precedence so that test plans can run in the proper order.

The only workaround for right now is to rename the folders to ensure the proper order. I am using numbers, but it's honestly a cludge. We should be able to reorder by drag-and-drop and not rely on the naming to accomplish the order.

**Comment by lanes100** [comment]
Bumping this. I have login calls that need to run before anything else and logout calls that need to run at the end of my collection runs and there is no way to do this without first making them a subfolder under a parent using 1 or Z as a workaround. Would really appreciate a fix for this.

**Comment by jamesvl** [comment]
I know "chime in" comments aren't always helpful, but I cannot really move my team over to Bruno without a feature like this.

We're coming from a large internal API collection where the organization of the folders matters. Would love to see this as well, even though I realize getting multi-level drag and drop just right for a UX perspective isn't easy.

**Comment by RohanTalip** [comment]
FYI, I was able to change the order of my collections manually on macOS by:
1. quitting the `bruno` app
1. making a **backup** copy of the `$HOME/Library/Application Support/bruno/preferences.json` file (in case I got something wrong)
1. editing the `lastOpenedCollections` list in the `$HOME/Library/Application Support/bruno/preferences.json` file (make sure it is still valid JSON, otherwise the list is cleared - see making a backup copy above)
1. starting the `bruno` app again

**Comment by petervanderdoes** [comment]
Maybe a new ticket is needed but would also love to see ordering within a folder.

**Comment by yuniet87** [comment]
Hello everyone. Firstly, I like Bruno a lot. Congratulations to the team for the effort and good work!
This feature is going to be very appreciated. Any progress on it?

**Comment by nishantgoel87** [comment]
Hello Everyone, many congratulations to the team for this great tool.
Please update on the availability of this feature

**Comment by sanjai0py** [comment]
We are working on it #3755

**Comment by ramki-bruno** [comment]
This will actually be fixed by #2554, the PR is bit old, but we are looking in to it.

**Comment by anusree-bruno** [comment]
This feature is available in v2.3.0. Closing the issue.

**Comment by lelabo-m** [comment]
I am on Bruno 3.2.2 (Windows) and can't seem to make it work. I don't see the capabilities shown by #2554. When I drag a folder, I can only put it inside another folder.

Edit: it seems possible if I aim correctly between two folders, but I had no visual clue or UI feedback that it existed.

**Comment by anusree-bruno** [comment]
@lelabo-m Thanks for reporting, We need to show the target line on drag and drop. Will get this fixed.

**Comment by brock-bouchard** [comment]
@anusree-bruno is there an issue we can follow for getting the target line back?

**Comment by anusree-bruno** [comment]
https://github.com/usebruno/bruno/issues/7748
